x-pack/platform/plugins/shared/ml/public/application/overview/components/anomaly_detection_overview.tsx

```diff
@@ -0,0 +1,143 @@
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FC } from 'react';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiButton, EuiButtonEmpty, EuiEmptyPrompt, EuiImage, useEuiTheme } from '@elastic/eui';
+import type { MlSummaryJobs } from '../../../../common/types/anomaly_detection_jobs';
+import { ML_PAGES } from '../../../locator';
+import adImage from '../../jobs/jobs_list/components/anomaly_detection_empty_state/anomaly_detection_kibana.png';
+import { usePermissionCheck } from '../../capabilities/check_capabilities';
+import { mlNodesAvailable } from '../../ml_nodes_check';
+import { useMlApi, useMlLocator, useMlManagementLocator } from '../../contexts/kibana';
+import { AnomalyDetectionEmptyState } from '../../jobs/jobs_list/components/anomaly_detection_empty_state/anomaly_detection_empty_state';
+
+export const AnomalyDetectionOverviewCard: FC = () => {
+  const [canGetJobs, canCreateJob] = usePermissionCheck(['canGetJobs', 'canCreateJob']);
+  const disableCreateAnomalyDetectionJob = !canCreateJob || !mlNodesAvailable();
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasADJobs, setHasADJobs] = useState(false);
+
+  const mlApi = useMlApi();
+  const mlLocator = useMlLocator();
+  const mlManagementLocator = useMlManagementLocator();
+
+  const loadJobs = useCallback(async () => {
+    setIsLoading(true);
+
+    try {
+      const jobsResult: MlSummaryJobs = await mlApi.jobs.jobsSummary([]);
+      if (jobsResult?.length > 0) {
+        setHasADJobs(true);
+      }
+    } catch (e) {
+      setIsLoading(false);
+    }
+  }, [mlApi]);
+
+  useEffect(() => {
+    loadJobs();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const redirectToMultiMetricExplorer = useCallback(async () => {
+    if (!mlLocator) return;
+
+    await mlLocator.navigate({
+      sectionId: 'ml',
+      page: ML_PAGES.ANOMALY_EXPLORER,
+    });
+  }, [mlLocator]);
+
+  const redirectToManageJobs = useCallback(async () => {
+    if (!mlManagementLocator) return;
+
+    await mlManagementLocator.navigate({
+      sectionId: 'ml',
+      appId: `anomaly_detection`,
+    });
+  }, [mlManagementLocator]);
+  const showEmptyState = !hasADJobs && canCreateJob && !disableCreateAnomalyDetectionJob;
+
+  const availableActions = useMemo(() => {
+    const actions: EuiButtonProps[] = [];
+    if (hasADJobs) {
+      actions.push(
+        <EuiButton
+          color="primary"
+          onClick={redirectToMultiMetricExplorer}
+          isDisabled={!canGetJobs}
+          data-test-subj="multiMetricExplorerButton"
+        >
+          <FormattedMessage
+            id="xpack.ml.overview.anomalyDetection.multiMetricExplorerButtonText"
+            defaultMessage="Multi-metric explorer"
```

Comment:
This should say 'Anomaly Explorer' as I think we decided not to change the page name.

Reply:
Updated here [`35d2e68` (#204290)](https://github.com/elastic/kibana/pull/204290/commits/35d2e68fce444803015b9622b0e7e341729ce7c9)